slick/__init__.py

```diff
@@ -44,7 +44,7 @@ def add_widget(self, widget):
 
 # Setup the main Flask app
 app = Slick(__name__)
-app.config.from_envvar('SLICK_CONFIG_FILE')
```

Comment:
Why are you removing this? This prevents custom locations for the config location, which is very important when launching a WSGI server using something like gunicorn or uwsgi.